lib/transloadit/rails/engine.rb

```diff
@@ -45,25 +45,38 @@ def self.configuration
       #
       # Returns the Transloadit authentication object.
       #
-      def self.transloadit
+      # options  - The Hash options used to refine the auth params (default: {}):
+      #            :max_size - The Integer maximum size an upload can have in bytes (optional).
+      def self.transloadit(options = {})
         Transloadit.new(
           :key      => self.configuration['auth']['key'],
           :secret   => self.configuration['auth']['secret'],
           :duration => self.configuration['auth']['duration'],
-          :max_size => self.configuration['auth']['max_size']
+          :max_size => options[:max_size] || self.configuration['auth']['max_size']
         )
       end
 
       #
       # Creates an assembly for the named template.
       #
+      # name    - The String or Symbol template name.
+      # options - The Hash options used to refine the Assembly (default: {}):
+      #           :steps    - The Hash with Assembly Steps (optional).
+      #           :max_size - The Integer maximum size an upload can have in bytes (optional).
       def self.template(name, options = {})
```

Comment:
> yeah, but def self.template(name, max_size: nil, **options) looks better :)

I absolutely agree with you, but this is a small price to pay to be compatible with older versions :) Thanks again for your pull request :)